ci-operator/step-registry/acm/tests/clc-create/README.md

```diff
@@ -0,0 +1,44 @@
+# acm-tests-clc-create-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+- [Requirements](#requirements)
+  - [Infrastructure](#infrastructure)
+  - [Environment Variables](#environment-variables)
+
+## Purpose
+
+To test the managed cluster creation feature of ACM.
```

Comment:
@dtthuynh could you provide info here for 

how many managed clusters will be created?
what clouds they will be created on?

and anything else that you think a user of this step should now about the clc-create test